### ISSUE TITLE
DatePicker component lost focus exception

### ISSUE THREAD

**Issue by jxcproject** [ant-design-blazor/ant-design-blazor#3848]
When the DatePicker component sets ChangeOnClose and clears content, losing focus will result in an error.
Reproduction steps:
1. Click on the component with the mouse to place the cursor after the last date value, then press Backspace on the keyboard to delete the content one by one until all content is deleted
2. When the mouse clicks on a blank area other than a component, causing the component to lose focus, the following exception will be reported

blazor.webassembly.js:1 crit: Microsoft.AspNetCore.Components.WebAssembly.Rendering.WebAssemblyRenderer[100]
      Unhandled exception rendering component: Nullable object must have a value.
System.InvalidOperationException: Nullable object must have a value.
   at System.Nullable`1[[System.DateTime, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].get_Value()
   at AntDesign.DatePickerBase`1.<OnBlur>d__239[[System.Nullable`1[[System.DateTime, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]], System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].MoveNext()
   at AntDesign.DatePicker`1.<OnBlur>d__9[[System.Nullable`1[[System.DateTime, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]], System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].MoveNext()
   at Microsoft.AspNetCore.Components.ComponentBase.CallStateHasChangedOnAsyncCompletion(Task task)
   at Microsoft.AspNetCore.Components.RenderTree.Renderer.GetErrorHandledTask(Task taskToHandle, ComponentState owningComponentState)

```
<DatePicker TValue="DateTime?" ChangeOnClose Value="@dateTime" />

@code {
    private DateTime? dateTime = DateTime.Now;
}

```

**Comment by ElderJames** [comment]
Thanks for contacting us @jxcproject , I can't reproduce the issue, could you please post a demo for that?

**Comment by jxcproject** [comment]
![screenshots](https://github.com/ant-design-blazor/ant-design-blazor/assets/8786702/36baaa1b-3131-42be-8b4b-a32495f68f02)
[BlazorApp16.zip](https://github.com/ant-design-blazor/ant-design-blazor/files/15243339/BlazorApp16.zip)

@ElderJames

**Comment by ElderJames** [comment]
Hello @jxcproject , this issue have been fixed in 0.19.0
